### ISSUE TITLE
High Contrast: Link control missing any indication it can be click in high contrast when rendered as a button

### ISSUE THREAD

**Issue by jascam** [OfficeDev/office-ui-fabric-react#2541]
### Bug Report

- __Package version(s)__: latest
- __Browser and OS versions__:
I have seen this in FireFox and Edge.  I am running windows 10 version 1703.

#### Priorities and help requested (not applicable if asking question):

Are you willing to submit a PR to fix? (Yes, No) No

Requested priority: (Blocking, High, Normal, Low) 
Normal

Products/sites affected: (if applicable)

#### Describe the issue:
The Link control will render as a button if there is no href provided.  In this rendering state if the user is in high contrast white or black the text is rendered normally without any indication that it is selectable.

#### Actual behavior:
Text renders without hint to user that it is a click target.

![image](https://user-images.githubusercontent.com/13953383/29436255-88971ea0-835f-11e7-84ae-a737c462859f.png)

#### Expected behavior:
button should look like a link since this is how it renders out of high contrast.

### If applicable, please provide a codepen repro:
Turn on High contrast white.
navigate to: https://dev.office.com/fabric#/components/link
Look at the link text
Notice that the 2'nd link does not have visual indication that it is selectable.
